pyEpiabm/pyEpiabm/core/person.py

```diff
@@ -214,12 +217,35 @@ def remove_person(self):
         Used to remove travellers from the population.
 
         """
-        self.microcell.cell.compartment_counter.\
+        self.microcell.cell.compartment_counter. \
             _increment_compartment(-1, self.infection_status,
                                    self.age_group)
-        self.microcell.compartment_counter.\
+        self.microcell.compartment_counter. \
             _increment_compartment(-1, self.infection_status,
                                    self.age_group)
         self.microcell.cell.persons.remove(self)
         self.microcell.persons.remove(self)
         self.household.persons.remove(self)
+
+    def set_id(self, id: str):
+        """Updates ID of person (i.e. for input from file).
+        ID format: 4.3.2.1 represents cell 4, microcell 3 within this cell,
+        household 2 within this microcell, and person 1 within this
+        household. The id will only be changed if there is a match.
```

Comment:
```suggestion
        household. The id will only be changed if there is a duplicate.
```


Apologies if this suggested change takes you over line length limits

Reply:
I don't quite understand this suggestion - the id will only be changed if it is of the correct format perhaps?